apps/gateway/lib/gateway/router.ex

```diff
@@ -15,6 +15,13 @@ defmodule Gateway.Router do
     get "/:provider/token/:token_id", Controllers.AuthController, :validate_token
   end
 
+  scope "/users", Gateway do
+    pipe_through :api
+
+    post "/users/:user_id/change_username/:username", Controllers.UserController, :change_username
+    post "/users/:user_id/change_profile_picture/:profile_picture", Controllers.UserController, :change_profile_picture
```

Comment:
Both endpoints should be either PUT or PATCH

Also, both of this endpoints could be the same `/users/:user_id` and `update_user_params` in the controller, the clients would send the field to modify in the body of the request

Reply:
Sure! Done!